### ISSUE TITLE
The jbox link is inaccessible

### ISSUE THREAD

**Issue by merlinarer** [ChrisAllenMing/GPA-detection#21]
The jbox link you provided is inaccessible, could you please update it?

**Comment by ChrisAllenMing** [comment]
Moved to Google Drive.

**Comment by merlinarer** [comment]
> Moved to Google Drive.

thanks a lot, dataset split you provided in the issue is also inaccessible, could you please provide it again? btw, there are 3 level fog densities (beta=0.005,0.01,0.02) in the foggycity, I would like to know which level you have used in your paper?

**Comment by ChrisAllenMing** [comment]
The dataset split can be accessed in this [Google Drive link](https://drive.google.com/drive/folders/1tzn_mblhQLQOcXzWhuNTynZt8LtoYof_?usp=sharing). We use beta=0.01 in our experiments.

**Comment by merlinarer** [comment]
> The dataset split can be accessed in this [Google Drive link](https://drive.google.com/drive/folders/1tzn_mblhQLQOcXzWhuNTynZt8LtoYof_?usp=sharing). We use beta=0.01 in our experiments.

Thank you for your quick reply~ another question is that kitti has 7481 images, sim10k has 10k images, why you filter out some of them to use in your exps other than all of them? Do you filter randomly?

**Comment by ChrisAllenMing** [comment]
Yeah, it's based on a random filtering on those images containing no cars.

**Comment by merlinarer** [comment]
> Yeah, it's based on a random filtering on those images containing no cars.

Hello,

The result that I reproduce on K2C is:
0.377->0.382->0.364->0.343…… 
the performance is decreasing with the epochs just as the the result reproduced by others in [issue](https://github.com/ChrisAllenMing/GPA-detection/issues/10).

I checkd the commands and I think everything has aligned with yours. I can also reproduce the source only result in the paper.

Would you please check your code bcause the code released is part of your project?  Or it is possible to release all the code because it's been a long time since CVPR 2020? Thank you ~

**Comment by kongyan66** [comment]
@merlinarer I just start to try this code and have some questions ,Could  i communicate with you privately, my QQ is 1042869685 .
